docs/source/rest-api/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _sdc-api:
+
+REST API
+========
+
+The SDC supports a REST API that allows users to upload and download files, as
+well as query for file metadata. The following documentation describes the
+various endpoints that are supported and how to use them.
+
+
+.. openapi:: openapi.yml
+   :group:
+   :include: /upload
+
+**Example Usage:**
+
+.. code-block:: bash
+
+   curl -X GET -H "Accept: application/json" https://api.dev.imap-mission.com/upload/imap_codice_l1a_lo_20260101_20260102_v01-01.cdf
+
+**Possible Responses:**
+
+.. code-block:: json
+
+   {"statusCode": 200, "body": "s3://sds-data/imap/codice/l1a/2026/01/imap_codice_l1a_lo_20260101_20260102_v01-01.cdf"}
+   {"statusCode": 400, "body": "A pre-signed URL could not be generated. Please ensure that the file name matches mission file naming conventions."}
+
+
+.. openapi:: openapi.yml
+   :group:
+   :include: /download
```

Comment:
Unfortunately I had to manually create some examples here. Currently the `sphinxcontrib-openapi` package does not automatically generate examples listed in the `openapi.yml` configuration successfully, there are some known issues with it. I guess the upside is that we will have finer control over what we want our users to see.

Reply:
We will want to move this onto the SDC's main website once that is up anyways, so I wouldn't spend too much time on these inconsistencies yet.